scripts/js/lib/api/saveImages.ts

```diff
@@ -13,6 +13,8 @@
 import { mkdirp } from "mkdirp";
 import pMap from "p-map";
 import { copyFile } from "fs/promises";
+import { extname } from "node:path";
```

Comment:
Nit: could you please group the imports in this file by std lib and third-party?